Resources/Prototypes/Entities/Clothing/Back/backpacks.yml

```diff
@@ -116,6 +116,51 @@
   - type: Clothing
     sprite: Clothing/Back/Backpacks/science.rsi
 
+
+- type: entity
+  parent: ClothingBackpack
+  id: ClothingBackpackVirology
+  name: virology backpack
+  description: A backpack made of hypo-allergenic fibers. It's designed to help prevent the spread of disease. Smells like monkey.
+  components:
+  - type: Sprite
+    sprite: Clothing/Back/Backpacks/virology.rsi
+  - type: Clothing
+    sprite: Clothing/Back/Backpacks/virology.rsi
```

Comment:
Should these be netsynced?

Reply:
Could you elaborate please as I'm not familiar? And I'll make all the changes discussed